config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Courtfinder::Application.routes.draw do
 
+  get 'admin/ping', to: 'home#ping', format: :json
```

Comment:
when combining this with https://github.com/ministryofjustice/courtfinder/pull/246 , wouldn't it make more sense to have the ping in the heartbeat controller?